play-frontend-hmrc-play-30/src/test/scala/uk/gov/hmrc/hmrcfrontend/views/helpers/hmrcInternalHeadSpec.scala

```diff
@@ -73,21 +71,7 @@ class hmrcInternalHeadSpec
       val content          = contentAsString(hmrcInternalHead())
 
       content should include regex
-        """<!--\[if gt IE 8\]><!-->
-          |<link href="/some-service/hmrc-frontend/assets/hmrc-frontend-\d+.\d+.\d+.min.css"  +media="all" rel="stylesheet" type="text/css" />
-          |<!--<!\[endif\]-->""".stripMargin.r
-    }
-
-    "include the hmrc-frontend minified ie8 css bundle and minified html5shiv bundle" in {
-      val hmrcInternalHead = app.injector.instanceOf[HmrcInternalHead]
-      hmrcfrontend.RoutesPrefix.setPrefix("/some-service/hmrc-frontend")
-      val content          = contentAsString(hmrcInternalHead())
-
-      content should include regex
-        """<!--\[if lte IE 8\]>
-          |<script src="/some-service/hmrc-frontend/assets/vendor/html5shiv.min.js"></script>
```

Comment:
Ah, just spotted that html5shiv is gone - I think we were going to keep this for a bit longer because without it the pta/bta/sca account menu breaks in IE8 - think some discussion around it relating to PLATUI-2684 in slack

Reply:
(this was because we're removing it and will tell maintainers of sca-wrapper and they can add it in if they want (or add a wrapping div to avoid needing extra js)